tests/console/gpg.pm

```diff
@@ -177,7 +177,15 @@ sub run {
         libgcrypt20 => '1.9.0',
         'libgcrypt20-hmac' => '1.9.0'
     };
-    zypper_call("in " . join(' ', keys %$pkg_list));
+    my $ret;
+    if (is_public_cloud() && check_var('BETA', '1')) {
+        ## We don't continue when latest PC image is behind repos
+        $ret = zypper_call("in " . join(' ', keys %$pkg_list), exitcode => [0, 4]);
+        record_info("X", $ret);
+        exit(1) if ($ret == "4");
```

Comment:
Let's use return/die/softfail as everywhere else in the test code. 

```
Don't use exit to abort a subroutine if there's any chance that someone might want to trap whatever error happened. Use die instead, which can be trapped by an eval.
```

IIRC, tests are eventually running in `eval`, but I am not that much experienced over here.